docs/Project.toml

```diff
@@ -10,3 +10,6 @@ Documenter = "1"
 DocumenterCitations = "1.3"
 DocumenterInterLinks = "1"
 LinearAlgebra = "1.11"
+
+[sources]
+LieGroups = { path = ".." }
```

Comment:
What does this thing do?

Reply:
Formerly we manually said that we add LieGroups.jl in development mode from the parent folder.

This is a bit easier and available since 1.11, and states that LieGroups.jl should be loaded from there. That way we do not have to do that anew manually every run.

Reply:
cf. https://github.com/JuliaLang/Pkg.jl/pull/3783#issuecomment-1935523553 or you can also easier add unregistered packages https://julialang.org/blog/2024/10/julia-1.11-highlights/#sources_section_in_projecttoml_in_pkgjl.

But here it is just an easier path to always load LieGroups.jl from parent folder